### PR TITLE
Parser add unRegister method

### PR DESCRIPTION
**Pull request by dongasai**
Add the unRegister method to unRegister

I had a situation in development where I needed to unregister but _options was private so I couldn't unregister, so I added this.

### CODE REVIEW

File: src/Input/Parser.php [hunk 1]
```diff
@@ -247,6 +247,28 @@ protected function register(Parameter $param)
...
     }
+    
+    /**
+     * unRegister a new argument/option.
+     * 
+     * @param string $name
+     */
+    public function unRegister($name)
+    {
+        if (isset($this->_values[$name])) {
```
Comment (adhocore) [inline src/Input/Parser.php:258]: for unset no need to check first with isset.

also multiple unset can be combined

eg: `unset($a['key1'], $a['key2']);`

**Comment by adhocore** [comment]
thank you very much. please try to get build pass somehow. 👏

**Comment by dongasai** [comment]
In the https://travis-ci.com/github/adhocore/php-cli/jobs/467713360 mission tasks fail because the configuration error

**Comment by adhocore** [comment]
thanks, instead of using unregister() method, can we rename it to unset()?

we have register() method with Parameter class as input arg which is not in line with unregister() method with string arg.

**Comment by dongasai** [comment]
@adhocore .

**Pull request merged**